e2e-tests-docker-compose.yml

```diff
@@ -35,7 +35,7 @@ services:
   registry:
     image: ${REGISTRY_IMAGE:-registry:2}
     ports:
-      - "5000:5000"
+      - "5001:5000"
```

Comment:
I made this change because macOS uses port 5000 for ControlCenter now...

Reply:
we need to change this port in tests/testcases/mod.rs as we use it there

Reply:
or may be not. because tests run in docker context as well